src/main/java/com/novi/springboottechiteasy/mappers/CiModuleDtoMapper.java

```diff
@@ -0,0 +1,34 @@
+package com.novi.springboottechiteasy.mappers;
+
+import com.novi.springboottechiteasy.dtos.cimoduledtos.CiModuleDto;
+import com.novi.springboottechiteasy.models.CiModule;
+import com.novi.springboottechiteasy.models.Television;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class CiModuleDtoMapper {
+    public CiModuleDto mapToDto(CiModule ciModule) {
```

Comment:
Waarom heb je in deze klasse geen "mapToEntity" methode? Je hebt die nu in de service staan, maar het grote voordeel van deze klasse is juist dat je de mapper methodes groepeert, dan moet je ook alle mapper methodes doen.